src/Http/Controllers/ContentTypes/Checkbox.php

```diff
@@ -9,6 +9,6 @@ class Checkbox extends BaseType
      */
     public function handle()
     {
-        return (int) $this->request->input($this->row->field, 0);
+        return (int) ($this->request->input($this->row->field) == 'on') ? true : false;
```

Comment:
testing with `==` will already return a boolean... why do you need the ternary operator returning `true` or `false`..?

Reply:
I changed it...